### ISSUE TITLE
Monolith Spawning issues (Anomaly Surveyor)

### ISSUE THREAD

**Issue by jrossignol** [jrossignol/ContractConfigurator#301]
The picture says it all:

![image](https://cloud.githubusercontent.com/assets/8717685/9450649/258f8406-4a67-11e5-9b4d-7eb0107097ef.png)

Credit to Enceos (and some others before him).

**Comment by jrossignol** [comment]
Willing to bet this is the issue:
```
[Part]: PartModule indexing mismatch at mega.monolith, index 0.
Node 'ModuleTripLogger' found in loaded data, but 'ModuleAGX' is defined in prefab.
Looking for ModuleTripLogger in other indices...

NullReferenceException: Object reference not set to an instance of an object
  at Part.ModulesOnStart () [0x00000] in <filename unknown>:0 
  at Part+.MoveNext () [0x00000] in <filename unknown>:0 
```

**Comment by jrossignol** [comment]
Also, PersistentRotation errors:
```
KeyNotFoundException: The given key was not present in the dictionary.
  at System.Collections.Generic.Dictionary`2[System.String,ITargetable].get_Item (System.String key) [0x00000] in <filename unknown>:0 
  at Rotate.VehicleRotator.FixedUpdate () [0x00000] in <filename unknown>:0 
```

**Comment by jrossignol** [comment]
Fixed for 1.7.0...  yay!
